src/client/src/OpenFin/utils/tileWindowNames.ts

```diff
@@ -0,0 +1,12 @@
+const tileWindowNames = new Set()
+tileWindowNames.add("EURUSD")
```

Comment:
This manual method of adding the windows is super brittle and will cause some hard to figure out bugs for anyone who adds another currency pair in the future. A better method might be to generate tileWindowNames from the currencyPair stream or something else that will update automatically with the backend.

Reply:
it actually exposes a bug in the window close logic (been there a while by the looks, and I didn't spot it either) so I'll rename these to "Reactive-Trader-FX-Spot-Tile--CCYCCY" in the upgrade branch and close these too when the main RT close is clicked

Reply:
> we should give the tile windows a better name than this, anyway, to identify each one as a Tile, with the currency suffixed, then we should not care about enumerating the currencies ..

I do agree the names could be better, they should have "tile" in the name. Right, then I can filter by a regex and won't need to touch the currencyPair observable at all.